introduction.md

```diff
@@ -1 +1,2 @@
-Hello. This is a simple introduction. My favourite path is "HTML & CSS". 
\ No newline at end of file
+Hello. This is a simple introduction. My favourite path is "HTML & CSS". 
+* HTML, CSS, GIT.
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?